modules/iam-service-account/iam.tf

```diff
@@ -101,6 +108,17 @@ resource "google_project_iam_member" "project-roles" {
   member  = local.resource_iam_email
 }
 
+resource "google_service_account_iam_member" "cicd-resman" {
+  provider = google-beta
```

Comment:
do you need the beta provider here?

Reply:
Maybe not, was playing it safe since we're using WIF principals/principal sets